node-cli/manageTeam.js

```diff
@@ -0,0 +1,62 @@
+const inquirer = require("inquirer");
+const Realm = require("realm");
+const index = require("./index");
+const main = require("./main");
+const output = require("./output");
+const users = require("./users");
+const team = require("./team");
+
+
+const Choices = {
+    GetTeamMembers: "Get my team members",
+    AddTeamMember: "Add a team member",
+    RemoveTeamMember: "Remove a team member",
+    MainMenu: "Return to main menu",
+    LogOut: "Log out / Quit",
+  };
+  
+  async function manageTeamMenu() {
+    try {
+      const answers = await inquirer.prompt({
+        type: "rawlist",
+        name: "manageTeamMenu",
+        message: "What would you like to do?",
+        choices: [...Object.values(Choices), new inquirer.Separator()],
+      });
+  
+      switch (answers.manageTeamMenu) {
+        case Choices.GetTeamMembers: {
+          await team.getTeamMembers();
+          return manageTeamMenu();
+        }
+        case Choices.AddTeamMember: {
+          await team.addTeamMember();
+          return manageTeamMenu();
+        }
+        case Choices.RemoveTeamMember: {
+          await team.removeTeamMember();
+          return manageTeamMenu();
+        }
+        case Choices.MainMenu: {
+          return main.mainMenu();
+        }
```

Comment:
AFAICT the only way to get to this menu is from the project menu. Can we add another menu option that returns you back to the project you were just looking at? (presumably the user's own project)